dimagi/utils/web.py

```diff
@@ -16,18 +20,19 @@
 from datetime import date, datetime, time
 from decimal import Decimal
 
+
 def get_url_base():
+    return '{}://{}'.format(settings.DEFAULT_PROTOCOL, get_site_domain())
+
+
+def get_site_domain():
     try:
-        protocol = settings.DEFAULT_PROTOCOL
+        return settings.BASE_ADDRESS
     except AttributeError:
-        protocol = 'http'
-    address = getattr(settings, "BASE_ADDRESS", None)
-    if address is None:
-        address = Site.objects.get(id = settings.SITE_ID).domain
-    return '%s://%s' % (protocol, address)
-
-def get_secure_url_base():
-    return 'https://%s' % Site.objects.get(id = settings.SITE_ID).domain
+        warnings.warn("Please don't use Site to set domain anymore; "
+                      "use BASE_ADDRESS instead.", DeprecationWarning)
+        return Site.objects.get(id=settings.SITE_ID).domain
```

Comment:
BASE_ADDRESS is also referenced [here](https://github.com/dimagi/commcare-hq/blob/e648657dc663cccea414da8df106a26e851b255b/corehq/apps/builds/README.md#make-commcare-use-this-ip-address) and in a [testrunner error message](https://github.com/dimagi/commcare-hq/blob/e648657dc663cccea414da8df106a26e851b255b/testrunner.py#L125) and in [localsettings.example.py](https://github.com/dimagi/commcare-hq/blob/e648657dc663cccea414da8df106a26e851b255b/localsettings.example.py#L128)